service/pipeline.ml

```diff
@@ -264,6 +280,7 @@ let build_with_cluster ~ocluster ~analysis ~lint ~master source =
     Node.branch ~label:"compilers" compilers;
     Node.branch ~label:"distributions" linux_distributions;
     Node.branch ~label:"macos" macos;
+    Node.branch ~label:"FreeBSD" freebsd;
```

Comment:
Otherwise it becomes first in the list and that's not really what most people should see first.
```suggestion
    Node.branch ~label:"freebsd" freebsd;
```